index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Aretha Franklin Bio</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+  <body>
+    <main id="main">
+      <h1 id="title">Aretha Franklin</h1>
+      <p>The woman who sangs Respect</p>
+      <figure id="img-div">
+        <img src="./img/Aretha.png" width="" alt="image of Aretha Franklin" id="image">
+        <figcaption id="img-caption">Aretha Franklin best singer of Blues in the World!</figcaption>
+      </figure>
+      <section id="tribute-info">
+        <p>Aretha Louise Franklin was born on March 25, 1942 (August 16, 2018) was an American singer, songwriter and pianist.[2] Referred to as the "Queen of Soul", she has twice been placed ninth in Rolling Stone's "100 Greatest Artists of All Time". With global sales of over 75 million records, Franklin is one of the world's best-selling music artists.
+        </p>
+        <a href="https://en.wikipedia.org/wiki/Aretha_Franklin" id="tribute-link" target="_blank">For more information visit this link</a>
+      </section>
+
+      <section class="timeline">
+        <h2 class="timelineTitle">Here is Aretha's time line life</h2>
+        <ul>
+          <li><strong>1914</strong> - Born in Cresco, Iowa</li>
+          <li><strong>1933</strong> - Leaves his family's farm to attend the University of Minnesota, thanks to a Depression era program known as the "National Youth Administration"</li>
+          <li><strong>1935</strong> - Has to stop school and save up more money. Works in the Civilian Conservation Corps, helping starving Americans. "I saw how food changed them", he said. "All of this left scars on me."</li>
+          <li><strong>1937</strong> - Finishes university and takes a job in the US Forestry Service</li>
+          <li><strong>1938</strong> - Marries wife of 69 years Margret Gibson. Gets laid off due to budget cuts. Inspired by Elvin Charles Stakman, he returns to school study under Stakman, who teaches him about breeding pest-resistent plants.</li>
```

Comment:
Improve the style of this section named timeline, you can review these resources : 

- https://developer.mozilla.org/en-US/docs/Web/CSS/text-align
- https://www.w3schools.com/cssref/pr_text_word-spacing.php
- https://developer.mozilla.org/en-US/docs/Web/CSS/line-height

Reply:
- [ ] As highlighted by the previous reviewer, kindly give reasons for requesting a change to the code base. Clear and specific reasons help the author of the code to understand the intent behind the suggested changes, fostering better communication and reducing ambiguity. This allows for a more constructive dialogue between team members. 👍 

#### Hint: Edit your comment above and give reasons why there is a need to remove unused code in the codebase. 😎